components/camel-google-bigquery/src/main/docs/google-summary.adoc

```diff
@@ -0,0 +1,28 @@
+= Google Component
+//THIS FILE IS COPIED: EDIT THE SOURCE FILE:
+:page-source: components/camel-google/src/main/docs/google-summary.adoc
```

Comment:
If this is really copied, it shouldn't be in this commit.  This also seems like a wrong location for it in any case. Also, is the summary-group attribute getting added? I thought that since we aren't using it any more we weren't adding it.

Reply:
@djencks yes, I will remove the use of summary-group as we can make use of docTitle. Also, we discussed generating the list on `summary.adoc` using the index-list, are you still planning to perform that in the future?